src/main/java/lotto/Lotto.java

```diff
@@ -0,0 +1,84 @@
+package lotto;
+
+import java.util.*;
+
+public class Lotto {
+    private static final Random random = new Random();
```

Comment:
아래에서 `random`을 사용할 필요 없을 것 같다고 의견 드리긴 했습니다만,
만약 사용하시더라도 `convention` 체크한번 부탁 드릴께요 😄 
```suggestion
    private static final Random RANDOM = new Random();
```